pkg/broker/ingress/multi_topic_decouple_sink.go

```diff
@@ -36,36 +37,40 @@ import (
 const projectEnvKey = "PROJECT_ID"
 
 // NewMultiTopicDecoupleSink creates a new multiTopicDecoupleSink.
-func NewMultiTopicDecoupleSink(ctx context.Context, options ...MultiTopicDecoupleSinkOption) (*multiTopicDecoupleSink, error) {
-	sink := &multiTopicDecoupleSink{
-		logger: logging.FromContext(ctx),
-	}
-
+func NewMultiTopicDecoupleSink(ctx context.Context, options ...MultiTopicDecoupleSinkOption) (sink *multiTopicDecoupleSink, err error) {
```

Comment:
I don't think I've seen us use the named return variables in the past. My preference would be to not use them.

Reply:
OK, I've removed them.